README.md

```diff
@@ -353,25 +394,25 @@ Look in:
 
 ### Hot Gym
 
-This is a simple example application that calls the SpatialPooler and
-TemporalMemory algorithms directly.  This attempts to predict the electrical
+This is a simple example application that calls the `SpatialPooler` and
+`TemporalMemory` algorithms directly.  This attempts to predict the electrical
 power consumption for a gymnasium over the course of several months.
 
 To run python version:
 ```
 python -m htm.examples.hotgym
 ```
 
-To run C++ version: (assuming current directory is top of repository)
-```
+To run C++ version: (assuming current directory is root of the repository)
```

Comment:
There is no C++ version of the hotgym example.

The names were changed to 'hello' and 'dynamic_hello' because they had nothing in common with the python hotgym example.